Asterisk/run.sh

```diff
@@ -51,10 +51,20 @@ bashio::var.json \
     -template /usr/share/tempio/http.conf.gtpl \
     -out /etc/asterisk/http.conf
 
+# Make a loop for these
+
 tempio \
     -template /usr/share/tempio/rtp.conf.gtpl \
     -out /etc/asterisk/rtp.conf
 
+tempio \
+    -template /usr/share/tempio/sip_custom.conf.gtpl \
+    -out /etc/asterisk/sip_custom.conf
+
+tempio \
+    -template /usr/share/tempio/sip_default.conf.gtpl \
+    -out /etc/asterisk/sip_default.conf
```

Comment:
I think you forgot to supply the configuration to tempio here.

Reply:
yes, I swapped it with `sip.conf`.